_board/bast_pro_mini_m0.md

```diff
@@ -12,9 +12,29 @@ features:
   - Breadboard-Friendly
 ---
 
-Is she the goddess of cats? 
+The Bast Pro Mini M0 features a powerful SAMD21E chip, an ARM Cortex-M0 based microcontroller with low power consumption and high performance, makes it ideal for endless applications.
 
-It’s Bast Pro Mini M0! With a powerful SAMD21E chip, an ARM Cortex-M0 based microcontroller with low power consumption and high performance, makes it ideal for endless applications.
+## Technical details
+
+* Microcontroller Arm Cortex-M0 ATSAMD21E18A with 48 MHz
+* Flash Memory 256 KB
+* SRAM Memory 32 KB
+* Operating Voltage	1.62 V – 3.63 V
```

Comment:
This is the chip voltage, not the board voltage. From 
https://electroniccats.com/store/bast-pro-mini-m0/

![image](https://user-images.githubusercontent.com/2847802/160407987-a1d81315-00c5-4cb1-b191-cb2c29726826.png)

Reply:
Updated, thanks for checking.